toolbox/ci2gubernator/lib/ci2gubernator.go

```diff
@@ -57,10 +57,10 @@ type Converter struct {
 }
 
 // NewConverter creates a Converter
-func NewConverter(bucket, org, repo, job string, build int) *Converter {
+func NewConverter(bucket, org, repo, job, mux string, build int) *Converter {
 	return &Converter{
 		gcsClient:     u.NewGCSClient(bucket),
-		gcsPathPrefix: fmt.Sprintf("%s/%d", job, build),
+		gcsPathPrefix: fmt.Sprintf("%s/%s/%d", mux, job, build),
```

Comment:
if mux is empty the resulting string will have two slashes. will that path work?

Reply:
Yes because mux is the first input to the format string so if it is empty there will only be a leading slash, and filepath.join will handle this. However, I changed it to use filepath.join directly instead of sprintf to make this case more evident